qiskit/extensions/quantum_initializer/ucrx.py

```diff
@@ -22,42 +19,51 @@
 a single-qubit rotation R_x(a_i) is applied to the target qubit.
 """
 import math
+from typing import List, Sequence, Union
 
 from qiskit.circuit.quantumcircuit import QuantumCircuit
-from qiskit.circuit.quantumregister import QuantumRegister
+from qiskit.circuit.quantumregister import QuantumRegister, Qubit
 from qiskit.exceptions import QiskitError
 from qiskit.extensions.quantum_initializer.uc_pauli_rot import UCPauliRotGate
 
+# Types that can be coerced to a valid Qubit specifier in a circuit.
+QubitSpecifier = Union[
```

Comment:
Can this be imported from `quantumcircuit.py` instead of duplicated here? 🙂

Reply:
+1 . Having is single copy of the type would be great. Otherwise, LGTM

Reply:
+1 . Having is single copy of the type would be great. Otherwise, LGTM